server/protocols/jmap/src/main/java/org/apache/james/jmap/json/ObjectMapperFactory.java

```diff
@@ -20,10 +20,15 @@
 package org.apache.james.jmap.json;
```

Comment:
Original changeset on **master** never pretended to know timezone:

```
    public ZonedDateTime getInternalDateAsZonedDateTime() {
            return ZonedDateTime.ofInstant(internalDate.toInstant(), UTC_ZONE_ID);
        }
```

It just convert it to UTC.